packages/code-du-travail-frontend/src/common/RelatedItems.js

```diff
@@ -1,165 +1,97 @@
-import {
-  getLabelBySource,
-  getRouteBySource,
-  SOURCES,
-} from "@socialgouv/cdtn-sources";
+import { getRouteBySource, SOURCES } from "@socialgouv/cdtn-sources";
 import {
   ArrowLink,
   Container,
   FlatList,
   Heading,
-  icons,
   theme,
 } from "@socialgouv/cdtn-ui";
 import Link from "next/link";
 import React from "react";
 import styled from "styled-components";
 
 import { matopush } from "../piwik";
-import { CallToActionTile } from "./tiles/CallToAction";
+
+const matoSelectRelated = (reco, selection) => {
+  matopush([
+    "trackEvent",
+    "selectRelated",
+    JSON.stringify({
+      reco,
```

Comment:
isn't reco always the same value here since you rmeove a / b testing ?

Reply:
nope, it can still come from either search or covisits (we use search if no covisits)